src/app/components/AutoComplete.svelte

```diff
@@ -0,0 +1,188 @@
+<script lang="ts">
+  // TODO: the following interface should be removed once #33 is done
+  interface dataObj {
+    bcp47Tag: string;
+    language: string;
+  }
+  enum keyboardKey {
+    Up = "ArrowUp",
+    Down = "ArrowDown",
+    Enter = "Enter",
+  }
+  export let label = "";
+  export let subtext = "";
+  // Testing Data
+  export let results: dataObj[] = [];
+  // To store Filted array
+  let filetred: dataObj[] = [];
+  // To store selected language
+  let selected: string = "";
+  // Toggle to show search list
+  let show = false;
+  // Index of Previous focus element
+  let prevIndex = -1;
+  // Index of focus element
+  let index = -1;
+
+  // Input to search
+  function onChange(event: any) {
```

Comment:
Try to avoid `any`; I think the type here is `Event`:

```suggestion
  function onChange(event: Event) {
```